content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -8,6 +8,10 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
+{{< tip >}}
+No special configuration is required to run Istio on clusters managed by Kops since version 1.22 running Kubernetes 1.22. For prior versions, you will need to continue to perform these steps.
+{{< /tip >}}
```

Comment:
I'm still slightly confused by this wording, especially what prior versions means. Is that prior Kubernetes version, or prior Kops versions?

This seems to say that if I am running Kubernetes 1.22 using Kops 1.22 I need no special configuration. What about using Kops 1.22 and running Kubernetes 1.21? This implies that it won't work. 

Do we have any ideas on what may work with Kops 1.23 when it's released? Will only k8s versions 1.23 and 1.22 need no special configuration with Kops 1.23? So this is really a new special configuration for Kubernetes clusters versions 1.22 and later (when using Kops)?

Reply:
This is more related to how kubernetes is setup prior to 1.22 rather than the default version specified in kops. You are free to use kops and have it manage and setup a kubernetes 1.18-1.22 cluster.